code/src/App.js

```diff
@@ -1,9 +1,25 @@
 import React from 'react'
 
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+
+import { rooms } from 'reducers/rooms'
+import { loader } from 'reducers/loader'
+import { GameWrapper } from './components/GameWrapper'
+
+// We have two recuders – the rooms, and the loader. The rooms is responsible for handling the game state, and the username.
+const reducer = combineReducers({ 
+  rooms: rooms.reducer,
+  loader: loader.reducer
+})
+const store = configureStore({ reducer });
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <Provider store={store}>
+      <main>
+      <GameWrapper/>
```

Comment:
Just a small note, be aware of indentation 😊

Reply:
👍